starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -50,3 +50,11 @@ public void findAll() {
     Assert.assertFalse(actualInventory.isEmpty());
   }
 }
+//task1
+public void createDAOtest() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    this.InventoryDAO.create(inventory);
+    Assert.assertFalse(actualInventory.isEmpty());
```

Comment:
In addition to checking an inventory is returned, we want to check against our original inventory (the one with constants) to see if they're the same **except** the ids.